tubular/scripts/retrieve_latest_base_ami.py

```diff
@@ -127,7 +127,8 @@ def _backoff_logger(details):
     max_tries=5, jitter=backoff.random_jitter, on_backoff=_backoff_logger,
 )
 def get_with_retry(url):
-    data = requests.get(url)
+    # Set timeout to 1 minute to ensure open connections get killed.
```

Comment:
I don't think the comment is needed, but it won't hurt (until someone changes the timeout and forgets to change the comment. 😉 )

Reply:
🙏 we delete this code before then.